docs/CeedlingPacket.md

```diff
@@ -985,6 +985,38 @@ that you execute on target hardware).
 
   **Default**: FALSE
 
+  ---
+
+  **Note:**
+
+    The configuration can be combined together with
+    
+    ```
+    :test_runner:
+        :cmdline_args: true
+    ```
+
+    After setting **cmdline_args** to **true**, the debuger will will execute each of test
+    case from crashing test file separatelly. The exclude and include test_case patterns will
+    be applied, to avoid or only execute exactly test case.
+
+    The .gcno and .gcda files will be generated and section of the code under test case
+    causing segmetation fault will be omitted from Coverage Report.
+
+    The default debugger (**gdb**)[https://www.sourceware.org/gdb/] can be switch by any other
+    via setting **GDB** and **GDB_ARGS** environment variables. At default set to:
```

Comment:
There is a `:tools:` section in the `project.yml` which should have these tools defined. Let's rather use that.

Reply:
Hi @Letme,

Will look at it, and try to apply your suggestion.

Reply:
Hi @Letme,

The configuration of the backtrace was moved under tool as you suggested:

'''
:tools:
        :backtrace_settings:
          :executable: gdb
          :arguments: [ "-q --eval-command run --eval-command backtrace --batch --args" ]
'''

Please let me know if this looks better right now for you.

Reply:
This looks indeed better. I added few more comments after a review of this part.